README.md

```diff
@@ -6,13 +6,101 @@
 
 This extension aims to support the creation of **Liquid templates** with the [Silverfin Templating Language](https://developer.silverfin.com/docs) and the development of [Liquid testing YAML](https://developer.silverfin.com/docs/liquid-testing) files.
 
+## Setup & Basic Usage
```

Comment:
I would add info on the prerequisites to start with here:
* Request API credentials from Silverfin (client ID / client secret)
* Link to sf_toolkit repository to follow the detailed setup steps (required) and install the CLI (optional)